RecoParticleFlow/PFProducer/interface/BlockElementImporterBase.h

```diff
@@ -26,6 +26,8 @@ class BlockElementImporterBase {
 
   const std::string& name() const { return _importerName; }
 
+  enum VetoMode { pfRecTrackCollection = 0, ticlSeedingRegion = 1, pfCandidateCollection = 2 };
```

Comment:
```suggestion
  enum class VetoMode { pfRecTrackCollection = 0, ticlSeedingRegion = 1, pfCandidateCollection = 2 };
```
from https://github.com/cms-sw/cmssw/pull/32291#discussion_r540382461
there may need to be a cast when reading from the config parameters, but the rest of the C++ should be more type safe this way